### ISSUE TITLE
Test editing templates

### ISSUE THREAD

**Issue by laurelfulford** [Automattic/newspack-theme#1381]
WordPress 5.8 comes with the ability to create and edit templates via block templates. 

This is opt-in for the theme using `add_theme_support( 'block-templates' )`, so it won't be immediately available in the Newspack theme on upgrade - because of this, I've marked this as a low priority, it doesn't need to be completed before 5.8 is released. 

Depending on how it works with the theme, it could be a handy thing make available, or it could be something to consider for a future iteration of the theme. 

More info: https://make.wordpress.org/core/2021/06/16/introducing-the-template-editor-in-wordpress-5-8/

**Comment by laurelfulford** [comment]
Closing this issue for now since we're unlikely to add this option to the current theme.
